DataCleaning#1

HaniffZaid opened the pull request:
first attempt data cleaning

HaniffZaid committed:
Add files via upload

HaniffZaid committed:
Add files via upload

HaniffZaid committed:
Update README.md

HaniffZaid committed:
Update README.md

HaniffZaid committed:
Update README.md

HaniffZaid committed:
Update README.md

kameerahariru commented on Data_Engineer_KoNoHa.ipynb:
```diff
@@ -0,0 +1,2775 @@
+              "25%        2.000000      1.000000\n",
+              "50%        2.000000      2.000000\n",
+              "75%        4.000000      2.000000\n",
+              "max       20.000000     30.000000"
```
If this statistical value result can be round off into two decimal values, it will be easier to read!
My eyes hurtin' jkjk

HaniffZaid replied:
ok ok ok

Nash887 commented on Data_Engineer_KoNoHa.ipynb:
```diff
@@ -0,0 +1,2775 @@
+          "data": {
+            "text/plain": [
+              "                              Location         Price Rooms  Bathrooms  \\\n",
+              "0                   KLCC, Kuala Lumpur  RM 1,250,000   2+1        3.0   \n",
```
Can change location dtypes from object to numeric

HaniffZaid replied:
ok sure can

Nash887 commented on Data_Engineer_KoNoHa.ipynb:
```diff
@@ -0,0 +1,2775 @@
+              "25%        2.000000      1.000000\n",
+              "50%        2.000000      2.000000\n",
+              "75%        4.000000      2.000000\n",
+              "max       20.000000     30.000000"
```
max 20bathrooms is impossible for a unit of house.

HaniffZaid replied:
yes indeed

Nash887 reviewed (commented):
Please look after my comments. Thank you saudara!

kameerahariru commented on Data_Engineer_KoNoHa.ipynb:
```diff
@@ -0,0 +1,2775 @@
+      "cell_type": "code",
+      "source": [
+        "# Drop missing value\n",
+        "df = df.dropna()"
```
Carpark null value alone comprises of 31% percent of overall data. Do you think we should drop the car park column? Kalau tak drop nak buat cemana?

HaniffZaid replied:
ok just drop. no problem.

kameerahariru commented on Data_Engineer_KoNoHa.ipynb:
```diff
@@ -0,0 +1,2775 @@
+              "53868          Condominium       1544.0   Fully Furnished  \n",
+              "53869          Condominium        650.0  Partly Furnished  \n",
+              "53876          Condominium       3973.0  Partly Furnished  \n",
+              "53882          Condominium       1313.0  Partly Furnished  \n",
```
Can you please add a new column, Price/sqft. I think that will be helpful for machine learning process as we can put this value as independent value.

HaniffZaid replied:
still searching for the code. chatgpt pls help.

kameerahariru reviewed (commented):
Please review my reviews boss

HaniffZaid committed:
Day 3 data cleaning

22/08/2024 1030am
Task Done:

✅ Label encoder
✅ separate Rooms and Store column
✅ drop Car Parks column
✅ add Price/Sqft column
✅ data clean from 53883 to 40345

HaniffZaid committed:
Delete 220824_DataClean.csv

HaniffZaid committed:
day 3 data clean

22/08/2024 1030am
Task Done:

✅ Label encoder
✅ separate Rooms and Store column
✅ drop Car Parks column
✅ add Price/Sqft column
✅ data clean from 53883 to 40345

HaniffZaid committed:
Rename Data_Engineer_KoNoHa.ipynb to 210824_DataClean.ipynb

HaniffZaid committed:
Delete README.md

HaniffZaid committed:
Add files via upload

HaniffZaid committed:
Delete README.md

HaniffZaid committed:
Add files via upload
